hosted/src/NodeAllocator.cc

```diff
@@ -28,6 +29,53 @@ int ebbrt::NodeAllocator::DefaultNumaNodes;
 std::string ebbrt::NodeAllocator::DefaultArguments;
 std::string ebbrt::NodeAllocator::DefaultNetworkArguments;
 
+std::string ebbrt::NodeAllocator::RunCmd(std::string cmd) {
```

Comment:
Seems like the `cmd` should be a const char *, since this function doesn't need ownership. This makes the interface more flexible